src/tester/test_unittest.py

```diff
@@ -1102,49 +1108,29 @@ def test_find_good_motions(self):
         result_no_match = self.cleaner.find_good_motions(events_no_match, cleaner.GOOD_MOTIONS)
         self.assertEqual(result_no_match, [])
 
-    @patch("src.cleaner.Cleaner.load_json_file")
-    @patch("src.cleaner.Cleaner.write_json_output")
-    @patch("src.cleaner.Cleaner.load_and_map_charge_names")
-    def test_process_single_case(self, mock_load_map, mock_write, mock_load):
-        mock_load.return_value = {
-            "code": "123",
-            "county": "test_county",
-            "party information": {
-                "defense attorney": "John Doe",
-                "defense attorney phone number": "555-1234",
-                "appointed or retained": "appointed"
-            },
-            "charge information": [
-                {"level": "Misdemeanor", "charges": "Charge1", "statute": "123", "date": "12/01/2023"}
-            ],
-            "other events and hearings": ["Motion To Suppress"],
-            "html_hash": "test_hash"
-        }
-        mock_load_map.return_value = {"Charge1": {"mapped_field": "mapped_value"}}
-
-        county = "test_county"
-        folder_path = "case_json_folder"
-        case_file = "case1.json"
-
-        self.cleaner.process_single_case(county, folder_path, case_file)
-
-        mock_load.assert_called_once()
-        mock_write.assert_called_once()
-
-        # Check that the output contains expected fields
-        output_data = mock_write.call_args[0][1]
-        self.assertTrue("case_number" in output_data)
-        self.assertTrue("charges" in output_data)
-        self.assertTrue("motions" in output_data)
-        self.assertTrue("defense_attorney" in output_data)
-        self.assertTrue("county" in output_data)
-        self.assertTrue("html_hash" in output_data)
-        self.assertTrue("attorney_type" in output_data)
-        self.assertTrue("earliest_charge_date" in output_data)
-        self.assertTrue("has_evidence_of_representation" in output_data)
-        self.assertTrue("parsing_date" in output_data)
-
-    @patch("os.listdir", return_value=["case1.json", "case2.json"])
+    def test_process_single_case(self):
+        county = "hays"
```

Comment:
Seems good that we were able to avoid the `@patch`-ing happening for this test 👍